app/main/tests/tests_views.py

```diff
@@ -1,11 +1,10 @@
 from http import HTTPStatus
```

Comment:
Should we stick those tests to projects directory?

Reply:
They aren't related to projects though, feels like a weird move.